docs/readmes/lte/ha_setup.md

```diff
@@ -0,0 +1,179 @@
+---
+id: ha_setup
+title: HA AGW Setup
+sidebar_label: HA AGW Setup
+hide_title: true
+---
+# HA AGW Setup
+
+The following document outlines the necessary steps to deploy and configure a
+Magma access gateway on AWS. This document also outlines configuring the AWS
+gateway to serve as a secondary to a primary gateway running at an edge site.
+
+## Deployment
+
+### Build AGW AMI
+
+Steps:
+
+1. Download packer onto your host machine at https://www.packer.io/downloads.html
+2. Run the following
+
+```
+[~] cd magma/orc8r/tools/packer
+[~/magma/orc8r/tools/packer] packer build -force \
+    -var "aws_access_key=YOUR_ACCESS_KEY" \
+    -var "aws_secret_key=Your_SECRET_KEY" \
+    -var "subnet=YOUR_SUBNET" \
+    -var "vpc=YOUR_VPC" \
+    debian-stretch-aws.json
+```
+
+YOUR_SUBNET and YOUR_VPC should specify an existing subnet and vpc on your AWS
+region. The choice of subnet and vpc won't affect the final box. These are the
+subnet/vpc which the box is launched into while building.
+
+The result should show
+
+```
+==> Builds finished. The artifacts of successful builds are:
+--> amazon-ebs: AMIs were created:
+us-west-1: ami-0f1c9db5a767a0296
+```
+
+### Deploy AGW AMI
```

Comment:
I wonder if adding screenshots here might help, alternatively, if we have a video to provide a walkthrough of this feature as well, it would be great.